WordPress/gutenberg
# Fix navigation editor

## Description
Fixes #28076.

The experimental navigation screen currently errors when trying to open it.

The `blockInformation` isn't defined straight away, but that's not handled gracefully. I imagine this wasn't noticed because in other editors ListView only renders when the List View dropdown is opened, compared to the navigation screen where ListView is always rendered and visible.

## How has this been tested?
Added an e2e test.

Manual testing
1. Enable the Navigation Screen experiment
2. Browse to the navigation screen, it should open
3. Try creating a menu

## Types of changes

Bug fix (non-breaking change which fixes an issue)



## Checklist:
